[project-base] implemented picture element (#836)
shopsys/shopsys: docs/upgrade/UPGRADE-unreleased.md

    - `Tests\ShopBundle\Acceptance\acceptance\PageObject\Front\CartPage`
    - `src/Shopsys/ShopBundle/Resources/views/Front/Content/Order/PromoCode/index.html.twig`

> - *(optional)* to implement default image sizes for individual devices width [#836](https://github.com/shopsys/shopsys/pull/836) you have to

Reviewer: We don't use optional anymore
```suggestion
- *(low priority)* to implement default image sizes for individual devices width [#836](https://github.com/shopsys/shopsys/pull/836) you have to
```

>     - update `/project-base/src/Shopsys/ShopBundle/Resources/config/images.yml` as has been changed in this [commit](https://github.com/shopsys/shopsys/pull/836/files#diff-6519f98eb70e3d78e0f9756083222ff3)

Reviewer: ```suggestion
    - update `src/Shopsys/ShopBundle/Resources/config/images.yml` as has been changed in this [commit](https://github.com/shopsys/shopsys/pull/836/files#diff-6519f98eb70e3d78e0f9756083222ff3)
```

>     - update all image macros in twig template `/project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Advert/box.html.twig` as shown below

Reviewer: ```suggestion
    - update all image macros in twig template `src/Shopsys/ShopBundle/Resources/views/Front/Content/Advert/box.html.twig` as shown below
```

    ```twig
    {{ image(advert, { size: advert.positionName }) }}
    ```

>     - in order to show chagnes properly for section `noticer` in `images.yml` you need to create banner in administration as **Image with link**.

Reviewer: ```suggestion
    - in order to show changes properly for section `noticer` in `images.yml` you need to create banner in administration as **Image with link**.
```

>     - remove all images for folders (if a folder with images exists)

Reviewer: ```suggestion
    - remove all images in folders (if a folder with images exists)
```

>     - remove all images for folders (if a folder with images exists)

Reviewer: There is no information why should I do that. Is it to regenerate the pictures?

>         - `project-base/web/content/images/product/default`

Reviewer: ```suggestion
        - `web/content/images/product/default`
```

>         - `project-base/web/content/images/product/list`

Reviewer: ```suggestion
        - `web/content/images/product/list`
```

>         - `project-base/web/content/images/sliderItem/default`

Reviewer: ```suggestion
        - `web/content/images/sliderItem/default`
```
